src/Handlers/vHandler.cpp

```diff
@@ -204,16 +204,17 @@ vHandler * getHandler(pugi::xml_node node) {
 		ret = new acSyntheticTurbulence;
     } else if (name=="Run") {
 		output("Skipping 'Run' element");
-    }else {
+        ret = new Callback();
```

Comment:
Why generic Callback? I think it should be NULL. If there is segfault because of it - this should be solved, where the segfault occurs.